src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  notification: string;
+  onSetNotification: React.Dispatch<React.SetStateAction<string>>;
+};
+
+export const Notification: React.FC<Props> = React.memo(({
+  notification,
+  onSetNotification,
+}) => {
+  const notificationDelay = setTimeout(() => onSetNotification(''), 3000);
+
+  const clickHandler = () => {
```

Comment:
```suggestion
  const handleClick = () => {
```
Use imperative verbs for function names. The name should answer the question "What to do?"